freyja_apps/ibeji_adapter/README.md

```diff
@@ -0,0 +1,102 @@
+# Ibeji Adapter Freyja Example Application
+
+This Freyja Example Application utilizes the [Ibeji Digital Twin Adapter](../../freyja_adapters/digital_twin/ibeji_adapter/) and an in-memory mock cloud connector adapter to show a minimal connected local example of how to retrieve data from the vehicle.
+
+## Build and Run
+
+To build and run the application, follow these steps:
+
+1. (Optional) If necessary, author configuration overrides for the [`InMemoryMockMappingClient`](https://github.com/eclipse-ibeji/freyja/tree/main/mapping_clients/in_memory_mock_mapping_client). Refer to the adapter README files for instructions on how to do this. This repository provides overrides in the [`.freyja`](../../.freyja/) directory that can be used with the [`mixed` sample provided by Ibeji](https://github.com/eclipse-ibeji/ibeji/tree/main/samples/mixed).
+
+1. Set the `$FREYJA_HOME` environment variable. If you are using the provided overrides, you can run the following command to set the variable:
+
+        export FREYJA_HOME={path-to-repo-root}/.freyja
```

Comment:
I prefer that we don't have the home directory under the repo directory.

Reply:
this was by design. it allows us to have config overrides defined in the repo and use them directly

Reply:
Hmmm.  I'm still not happy with this, as the default should be related to where it is installed, not developed.  I will let this go for now.  We should fix this when we have a better install story.